frontend/src/views/Game.tsx

```diff
@@ -251,14 +276,14 @@ function GamePage() {
           // If this user refreshed and has already submitted code, load and save their latest code
           res.players.forEach((player) => {
             if (player.user.userId === location.state.currentUser.userId && player.code) {
-              setDefaultCodeFromProblems(res.problems, player.code, player.language as Language);
+              setDefaultCodeFromProblems(res.problems, player.code, [player.language as Language]);
```

Comment:
just a heads up, i don't think the player code is loaded after a refresh (this might be a later issue tho, especially since we'll have to figure out saving multiple submissions/code for different problems and everything)

Reply:
Yeah, I was quite certain that these lines were going to cause a problem. The alternative is saving another code array and language array in each player object, but I think there would be a problem with generating an incomplete array from the backend. Design-wise maybe we should just not remember submitted code after a refresh.

Reply:
yeah just thinking about it it seems like it's going to get too messy. I would prefer to remember submitted code, but at the same time there's a chance in my Redux PR that after refreshing you're booted from the room so who knows lol

However I would say that code should be remembered for each problem and each language when not refreshing (so if you just switch randomly between different problems and languages it won't lose your code)